README.md

```diff
@@ -1,68 +1,873 @@
-  ```bash
-  [path]$ git clone git@github.com:okta/okta-signin-widget.git && cd okta-signin-widget
-  ```
-
-2. [Install Bundler](http://bundler.io/) if you don't already have it.
 
-  ```bash
-  [path/okta-signin-widget]$ gem install bundler
-  ```
+    ```bash
+    $ git clone git@github.com:okta/okta-signin-widget.git && cd okta-signin-widget
+    ```
 
-3. Install our Ruby dependencies.
+2. [Install Bundler](http://bundler.io/) if you don't already have it, and then install our Ruby dependencies.
 
-  ```bash
-  [path/okta-signin-widget]$ bundle install
-  ```
+    ```bash
+    [okta-signin-widget]$ gem install bundler
+    [okta-signin-widget]$ bundle install
+    ```
 
-4. Install our Node dependencies.
+3. Install our Node dependencies.
 
-  ```bash
-  [path/okta-signin-widget]$ npm install
-  ```
+    ```bash
+    [okta-signin-widget]$ npm install
+    ```
 
-5. Create a `.widgetrc` file in the ```okta-signin-widget/``` directory. Add a base URL entry in `.widgetrc`.
+4. Create a `.widgetrc` file in the `okta-signin-widget` directory with an entry for `baseUrl`.
 
-  ```javascript
+    ```javascript
     {
       "widgetOptions": {
         "baseUrl": "https://your-org.okta.com"
       }
     }
-  ```
+    ```
 
-6. Build the widget, start a local connect server that hosts it, and launch a browser window with the widget running.
+5. Build the widget, start a local connect server that hosts it, and launch a browser window with the widget running.
 
-  ```bash
-  [path/okta-signin-widget]$ npm start
-  ```
+    ```bash
+    [okta-signin-widget]$ npm start
+    ```
 
-7. Finally, enable CORS support for our new server by [following these instructions](http://developer.okta.com/docs/guides/okta_sign-in_widget.html#configuring-cors-support-on-your-okta-organization). You can now authenticate to Okta using your very own, customizable widget!
+6. Finally, enable CORS support for our new server by [following these instructions](http://developer.okta.com/docs/guides/okta_sign-in_widget.html#configuring-cors-support-on-your-okta-organization). You can now authenticate to Okta using your very own, customizable widget!
 
 ## The `.widgetrc` config file
 
 The `.widgetrc` file is a configuration file that saves your local widget settings.
 
 | Property | Description |
 | --- | --- |
-| **widgetOptions** | Options that are passed to the widget on initialization - reference the [developer docs](http://developer.okta.com/docs/guides/okta_sign-in_widget.html#customizing-widget-features-and-text-labels-with-javascript) to see a complete list of configurable features. |
-| **serverPort** | The port the local server runs on. Defaults to *3000* |
+| **widgetOptions** | Config options that are passed to the widget on initialization. |
+| **serverPort** | The port the local server runs on. Defaults to `3000` |
 
-## Build and test commmands
+## Build and test commands
 
 | Command | Description |
 | --- | --- |
```

Comment:
Ah, this is how github markdown works for tables (need the separator row for it to show up as a table)